packages/cadl-rlc-test/package.json

```diff
@@ -7,7 +7,8 @@
     "@cadl-lang/ts-rlc": "0.0.1",
     "@cadl-lang/rest": "latest",
     "@cadl-lang/openapi": "latest",
-    "@azure-tools/cadl-autorest": "latest"
+    "@azure-tools/cadl-autorest": "latest",
```

Comment:
Use 'latest' may cause breaking change with the latest version of CADL.

Reply:
Currently we are still in feature branch `cadl-integration` in dev mode so we want to keep latest. We will update to fixed versions when merging back to main branch.